src/onboarding/onboarding-create-user.ts

```diff
@@ -81,6 +83,10 @@ class OnboardingCreateUser extends LitElement {
 
   protected firstUpdated(changedProps: PropertyValues) {
     super.firstUpdated(changedProps);
+    computeDirectionStyles(
+      translationMetadata.translations[this.language!].isRTL,
+      this
+    );
```

Comment:
This should be done at the parent level (`ha-onboarding`), not on the individual elements. And that uses the localize mixin, so this can be removed.